lib/global_config_service.rb

```diff
@@ -1,6 +1,6 @@
 class GlobalConfigService
   def self.load(config_key, default_value)
-    config = ENV.fetch(config_key) { GlobalConfig.get(config_key)[config_key] }
+    config = ENV.fetch(config_key, '').presence || GlobalConfig.get(config_key)[config_key]
```

Comment:
Note : To check whether we need the ENV.fetch()  here . in the local environment  since i have our .env.example copied over, i had empty environment variables for the slack config.   based on old code this will take precedence and won't return the global config values.

Reply:
Decided to remove the ENV.fetch here and rely on our fall back. 
cc: @pranavrajs